client/src/components/Review/style.css

```diff
@@ -0,0 +1,24 @@
+.review {
+  display: flex;
```

Comment:
This style is useless 
The div tag is block

Reply:
Done